cmd/backup_create.go

```diff
@@ -42,11 +42,13 @@ func backupCreateCommand(l log.Logger, conf config.Optimus, datastoreRepo models
 	backupCmd.Flags().BoolVar(&ignoreDownstream, "ignore-downstream", ignoreDownstream, "Do not take backups for dependent downstream resources")
 
 	backupCmd.RunE = func(cmd *cli.Command, args []string) error {
+		var err error
+		dsRepo := models.DatastoreRegistry
```

Comment:
It would be loaded through blank identifier import
```
_ "github.com/odpf/optimus/ext/datastore"
```
you can see it on `main.go`, it should be calling initialization of `models.DataRegistry` from [here](https://github.com/odpf/optimus/blob/a5349fe3f80f176d3ebece529c4c0fe582c00ba3/ext/datastore/bigquery/bigquery.go#L183), but I need to recheck first if it's working

Reply:
confirmed, model.DataRegistry will be loaded whenever we run optimus